src/user/views/audit_views.py

```diff
@@ -249,11 +251,11 @@ def contributions(self, request):
 
     @action(detail=False, methods=["post"])
     def flag(self, request):
-        moderator = request.user
+        hub_editor = request.user
```

Comment:
I understand that `editor` is a better name than `moderator` here but it could be that a moderator who is not an editor can also flag. Maybe we just say user?

But then again, what is this endpoint used for? If it is used for flagging then we should ensure anyone can flag and not just editors or moderators